Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -671,7 +708,6 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
       auto& trackIndex = primVerGIs[ti];
```

Comment:
I think it is better to use `TrackExtraInfo extraInfoHolder;` local to the loop, then you don't need all these reinitializations

Reply:
Yes, it is a better way indeed